static/usr/lib/systemd/system/tee-supplicant.service.d/core-override.conf

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Adapted from https://github.com/OP-TEE/optee_client/blob/6486773583b5983af8250a47cf07eca938e0e422/tee-supplicant/tee-supplicant%40.service.in
+[Unit]
+DefaultDependencies=no
+BindsTo=dev-teepriv0.device var-lib-optee\x2dclient-data-tee.mount
+After=dev-teepriv0.device var-lib-optee\x2dclient-data-tee.mount
+
+[Service]
+Type=notify
+ExecStartPre=-/usr/sbin/modprobe -v -r tpm_ftpm_tee
+ExecStartPost=-/usr/sbin/modprobe -v tpm_ftpm_tee
```

Comment:
Not sure why we need that. So you want to remove `tpm_ftpm_tee` before starting. Then once `tee-supplicant` has started and sent `READY=1`, reload the module.